metadata-db-app/src/cmr/metadata_db/data/generic_documents.clj

```diff
@@ -6,6 +6,10 @@
 
   (generate-concept-id [db document] "")
 
+  (get-concept-id
```

Comment:
This function does nothing and isn't marked as a stub, it should throw or be removed.

Reply:
I don't understand, it is used in cmr.metadata-db.data.oracle.generic-documents

Reply:
we talked, this file is a protocol, resolving conversation.